palace/CMakeLists.txt

```diff
@@ -112,6 +112,15 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 # Find MPI
 find_package(MPI REQUIRED)
 
+
+if(PALACE_WITH_SUNDIALS AND NOT PALACE_BUILD_EXTERNAL_DEPS)
+  # Spack's sundials requires MPI with C bindings, so we alias
+  # MPI_C to MPI_CXX (if it does not already exist).
+  if(NOT TARGET MPI::MPI_C AND TARGET MPI::MPI_CXX)
+    add_library(MPI::MPI_C ALIAS MPI::MPI_CXX)
+  endif()
+endif()
```

Comment:
Odd that this isn't the `find_package` default behavior given the MPI C++ bindings were deprecated in 3.0 - https://www.mpi-forum.org/docs/mpi-2.2/mpi22-report/node32.htm

Reply:
Interesting! I am not sure about this. We use MPI_CXX in a couple of other places